backend/ops_api/ops/schemas/cans.py

```diff
@@ -95,6 +95,13 @@ class FundingBudgetSchema(Schema):
     updated_by_user = fields.Nested(SafeUserSchema(), allow_none=True)
 
 
+class CreateUpdateFundingBudgetSchema(Schema):
```

Comment:
should we update the `openapi` schema as well?

Reply:
Yes this is a good point. I definitely should update the openapi and add the schema and endpoints.

Reply:
Done in my latest changelist.